### DISCUSSION TITLE
Hello team, in 3 node cluster getting  -context deadline exceeded (Client.Timeout exceeded while awaiting headers

### DISCUSSION THREAD

**Discussion by frootima** [grafana/mimir#6155] [General]
ts=2023-09-28T20:50:39.815032569Z caller=reporter.go:175 level=info msg="failed to send anonymous usage stats report" err="send the report to the stats server: Post \"https://stats.grafana.org/mimir-usage-report\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"
ts=2023-09-28T20:50:50.842474843Z caller=multitenant.go:528 level=info component=MultiTenantAlertmanager msg="synchronizing alertmanager configs for users"
ts=2023-09-28T20:51:05.842431861Z caller=multitenant.go:528 level=info component=MultiTenantAlertmanager msg="synchronizing alertmanager configs for users"
ts=2023-09-28T20:51:20.841719311

**Comment by charleskorn** [comment]
The error you've posted there is for Mimir's [anonymous usage stats reporting](https://grafana.com/docs/mimir/latest/configure/about-anonymous-usage-statistics-reporting/). It can safely be ignored, as it could be caused by a firewall or other network configuration blocking outbound connections to the stats server.

Are you seeing any other issues with Mimir?
